.github/workflows/backend-integration-test.yml

```diff
@@ -102,7 +98,7 @@ jobs:
           -x :web:test -x :clients:client-python:test -x :flink-connector:flink:test -x :spark-connector:test -x :spark-connector:spark-common:test 
           -x :spark-connector:spark-3.3:test -x :spark-connector:spark-3.4:test -x :spark-connector:spark-3.5:test 
           -x :spark-connector:spark-runtime-3.3:test -x :spark-connector:spark-runtime-3.4:test -x :spark-connector:spark-runtime-3.5:test
-          -x :authorizations:authorization-ranger:test
+          -x :authorizations:authorization-ranger:test -x :authorizations:authorization-ranger:test -x :web:integration-test:test -x :trino-connector:integration-test:test -x :trino-connector:test
```

Comment:
The parameter `skipWebITs` also can be removed and using `-x :web:integration-test:test` instead

Reply:
fixed